content/articles/url-record.markdown

```diff
@@ -25,6 +25,15 @@ Enter the hostname you would like to redirect from, and the URL to redirect to.
 
 Right now it is not possible for our redirection service to handle SSL requests for your domain names. Take a look at the article [Can a URL record redirect requests over SSL?](/articles/url-redirect-ssl) for additional information and an explanation of alternatives.
 
-## URL Rewrites
+## URL to URL redirects
+
+URL to URL redirects can't be done with our URL record. Only domains or subdomains can be redirected to complete URLs. 
+
+Example: foo.com/blog to bar.com/blog
+
+## Wildcard subdomain to URL redirects
+
+Is not possible to redirect wildcard subdomains to URLs
+
+Example: *.foo.com to bar.com/$1
```

Comment:
This line is partially incorrect, as we do support wildcard redirects but we don't support redirects based on matches. I'd rephrase the section to

```
## Match-based URL redirects

You currently cannot configure a redirect target that includes a back-reference (match) from the redirect source. For example, the following redirects will not work:

    *.foo.com to bar.com/$1
    *.foo.com to $1.bar.com
```

I also indented the example to appear as code.